src/index.js

```diff
@@ -106,8 +110,13 @@ function config(loader, appHost, configModuleId) {
 }
 
 function run() {
-  return ready(window).then(doc => {
-    initialize();
+  return ready().then((isNode: boolean) => {
+    if (!isNode) {
+      console.log(`initializing!`)
```

Comment:
Is the plan to let the consumer configure the PAL when importing the boostrapper? Right now it seems it's a hard dependency on `aurelia-pal-browser`? Then we only need a dependency on the `pal`. Does that make sense? I don't know if it's possible right now but I think that was how we thought about it long time ago @EisenbergEffect